spec/factories/loans.rb

```diff
@@ -0,0 +1,15 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :loan do
+    name { Faker::Company.buzzword }
```

Comment:
@gabrielgaraujo I tried to use `@name` in the spec file, but I kept getting undefined method. I wanted to replace the hard coded text for name.

Reply:
Everything looks great, lets check this issue you are facing together!